Makefile.pre.in

```diff
@@ -1681,15 +1681,18 @@ PYTHON_HEADERS= \
 		$(srcdir)/Include/codecs.h \
 		$(srcdir)/Include/compile.h \
 		$(srcdir)/Include/complexobject.h \
+		$(srcdir)/Include/datetime.h \
```

Comment:
This is for the datetime C API capsule. Perhaps this dependency should be more targeted.